editor/src/core/shared/comment-flags.ts

```diff
@@ -97,9 +97,9 @@ function getUtopiaCommentFlag(c: Comment, type: UtopiaCommentFlagType): UtopiaCo
     return null
   }
 
-  function parseIntOrNull(value: string): number | null {
+  function parseNonNegativeIntOrNull(value: string): number | null {
```

Comment:
Minor: But I think this rename makes sense.
```suggestion
  function parsePositiveInt(value: string): number | null {
```